Mr.Mulles new propeller-motionblur

SnailPerson opened the pull request:
Added _code_ not cote

SnailPerson committed:
Updated bell47g textures to support motionblur

SnailPerson committed:
Updated prop-textures to support motion blur

-> All three propeller variants now have a motion-blur texture which will appear at a certain speed

SnailPerson committed:
Updated prop-models to support motion blur

-> Propeller-blur support for small2blade, large2blade, small3blade, heli-rotor

SnailPerson committed:
Added cote to support new prop-blur animation

-> Propeller-blur support for small2blade, large2blade, small3blade, heli-rotor
-> Tweaked some exhaust particle settings for franklinengine

DonBruce64 reviewed (changes requested):
Please make the propellerlarge2blade texture a power of 2.  It makes the GPU process it better and faster.

SnailPerson committed:
Increased propellerlarge2blade.png res. to 128x128

SnailPerson committed:
Updated UV map

Updated uv map for propellerlarge2blade.obj to be compatible with the new 128x128 resolution

SnailPerson committed:
Increased resolution to 128x128p

from 132 because me not function correctly at 04:30 am

SnailPerson committed:
Updated Uv map, again

æ
